worldengine/cli/main.py

```diff
@@ -381,7 +393,7 @@ def main():
     sys.setrecursionlimit(args.recursion_limit)
 
     if args.number_of_plates < 1 or args.number_of_plates > 100:
-        usage(error="Number of plates should be in [1, 100]")
+        usage(error="Number of plates should be a in [1, 100]")
```

Comment:
No, I think the changes are part of the commit. I tried to checkout the branch from his repository and rebase it on origin/master but that did not work. I had to edit history, rebasing interactively and picking single changes from the first commit.